src/app/child-dev-project/attendance/add-day-attendance/roll-call/roll-call.component.spec.ts

```diff
@@ -177,4 +177,14 @@ describe("RollCallComponent", () => {
 
     expect(component.complete.emit).toHaveBeenCalledWith(note);
   }));
+
+  it("should complete roll call after clicking on save button", () => {
```

Comment:
This tests is failing now due to the changes to the `save` function

Reply:
Yes, It needs to be fixed now.